### PR TITLE
Party handling

### PR DESCRIPTION
**Pull request by PabloSebasVivanco**
Let's goooo!!

### CODE REVIEW

**Comment by andrewjunggg** [comment]
Items component and its features are still broken. Do not change the code base that is already on main. Please fix and test everything.

**Review by andrewjunggg — CHANGES_REQUESTED** [review]
PR still doesn't work on my end. Items component still isn't working properly.

**Comment by andrewjunggg** [comment]
I'm still getting errors when splitting the bill. Please test and fix.

File: src/components/SplitParty/SplitParty.js [hunk 1]
```diff
@@ -6,15 +6,16 @@ const SplitParty = (props) => {
...
+    setCalculatedSplit(splitAmount);
   };
 
   return (
     <React.Fragment>
       <div className="backdrop">
         {calculatedSplit === 0 ? (
           <div className="content">
-            Enter the amount of people in the party:
+            Calculated Enter the amount of people in the party:
```
Comment (andrewjujun) [inline src/components/SplitParty/SplitParty.js]: I'm not sure why you added "Calculated" here. Can you remove it please.

File: src/components/Board/Board.js [hunk 2]
```diff
@@ -56,16 +93,36 @@ const Board = () => {
         />
         <h1>Bill Splitter</h1>
       </header>
+     
       <NewItem onAddItems={addItemHandler} />
-      <Items datas={items} remove={removeItem} />
+      
+      {/* Check if items are available before rendering Items component */}
+      <div className="items">
```
Comment (andrewjujun) [inline src/components/Board/Board.js]: I don't think this is necessary. It's already handled on the Items component. so you just need to call `<Items datas={items} remove={removeItem} />`.

File: src/components/Board/Board.js [hunk 3]
```diff
@@ -26,13 +26,7 @@ const Board = () => {
...
   }
   
-  const [items, setItems] = useState(DEFAULT_ITEMS);
-
-  const addItemHandler = (item) => {
-    setItems((prevItems) => {
-      return [...prevItems, item];
-    });
-  };
+  const [items, setItems] = useState([]);
```
Comment (andrewjujun) [inline src/components/Board/Board.js]: Why do you change DEFAULT_ITEMS to `[]`?

File: src/components/Parties/Parties.js
Comment (andrewjujun) [inline src/components/Parties/Parties.js:1]: You can remove this file if it's not needed

File: src/components/NewParty/NewParty.js
Comment (andrewjujun) [inline src/components/NewParty/NewParty.js:1]: You can remove this file as well since it's not needed.

File: src/components/NewParty/NewPartyForm.js
Comment (andrewjujun) [inline src/components/NewParty/NewPartyForm.js:1]: You can remove this file since this is not needed.

**Review by andrewjunggg — CHANGES_REQUESTED** [review]
Please make another fix on some of the things that I mentioned.

File: src/components/SplitParty/SplitParty.js (hunk 1 above)
Comment (PabloSebasVivanco) [reply src/components/SplitParty/SplitParty.js]: idk why is there
I think I was adding another line on top but I erased it on my local it was just for styling.
I fixed now

File: src/components/Board/Board.js (hunk 2 above)
Comment (PabloSebasVivanco) [reply src/components/Board/Board.js]: Got it

File: src/components/Parties/Parties.js
Comment (PabloSebasVivanco) [reply src/components/Parties/Parties.js:1]: Erased

File: src/components/NewParty/NewPartyForm.js
Comment (PabloSebasVivanco) [reply src/components/NewParty/NewPartyForm.js:1]: ok

**Review by andrewjunggg — APPROVED** [review]
Great job

**Pull request merged**